Src/Sentry.Xamarin/Extensions/SentryXamarinOptionsExtensions.cs

```diff
@@ -112,7 +112,7 @@ internal static bool RegisterNativeIntegrations(this SentryXamarinOptions option
 
         internal static bool RegisterNativeActivityStatus(this SentryXamarinOptions options)
         {
-#if NATIVE_PROCESSOR
+#if LIFECYCLE_PROCESSOR
```

Comment:
Just wondering why you made this change?

Reply:
It's explained in my original pull request comment above:

Session logging is not implemented for Xamarin.Mac so an additional constant was added (LIFECYCLE_PROCESSOR) to for iOS, Android and UWP that do have that functionality. Previously, NATIVE_PROCESSOR was an indication that there were native implementations for both session logging and event processing.

Reply:
On the Mac, we don't need to keep track of when the app is backgrounded, or brought to the front (at least in my opinion).

Reply:
Fair enough